IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json

```diff
@@ -229,6 +229,116 @@
 						}
 					]
 				},
+				{
+					"name": "Change of the Business Date or Current Branch",
+					"request": {
+						"method": "PUT",
+						"header": [
+							{
+								"key": "Accept",
+								"type": "text",
+								"value": "application/json"
+							},
+							{
+								"key": "Content-Type",
+								"type": "text",
+								"value": "application/json"
+							},
+							{
+								"key": "PX-CbApiBusinessDate",
+								"type": "text",
+								"value": "2018/01/01"
+							},
+							{
+								"key": "PX-CbApiBranch",
+								"type": "text",
+								"value": "SweetLife Store"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"Description\": {\"value\": \"Test transaction description\"}}"
+						},
+						"url": {
+							"raw": "{{EndpointURL}}JournalTransaction",
+							"host": [
+								"{{EndpointURL}}JournalTransaction"
+							]
+						},
+						"description": "This example shows how to set the active branch and business date for a request. This process is described in [Change of the Business Date or Current Branch](https://help.acumatica.com/Help?ScreenId=ShowWiki&pageid=9821cff9-4970-4153-a0f8-dbf5758133a7)."
+					},
+					"response": [
+						{
+							"name": "New journal transaction with changed business date and branch",
```

Comment:
Please change this name to the name of the example in help (Change of the Business Date or Current Branch)